resource/schema/numberdefault/static_value_test.go

```diff
@@ -0,0 +1,45 @@
+package numberdefault_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/defaults"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/numberdefault"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func TestStaticValueDefaultNumber(t *testing.T) {
```

Comment:
Nit: Should this be `TestStaticBigFloat` to match the function name?

Reply:
I've updated the naming of the `/resource/schema` static default test functions for primitives to make them consistent.